standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/conf/MetastoreConf.java

```diff
@@ -1808,6 +1808,27 @@ public enum ConfVars {
         "hive.metastore.properties.servlet.auth", "jwt",
         "Property-maps servlet authentication method (simple or jwt)."
     ),
+    PROPERTIES_NS("hive.metastore.properties.namespaces",
```

Comment:
why do you put rest-catalog properties inside of HMS? shouldn't it have it's own configs?

Reply:
Because HMS starts the HMS-Catalog service.

Reply:
why in a world?